--- old/src/CMakeLists.txt
+++ new/src/CMakeLists.txt
@@ -152,21 +152,7 @@
 include(FetchContent)

 find_package(Git)
-if(GIT_FOUND)
-  FetchContent_Declare(
-    clone_strings
-    GIT_REPOSITORY    https://github.com/aseprite/strings.git
-    GIT_TAG           origin/main
-    SOURCE_DIR        ${DATA_OUTPUT_DIR}/strings.git
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND     ""
-    INSTALL_COMMAND   ""
-    TEST_COMMAND      "")
-  FetchContent_MakeAvailable(clone_strings)
-  add_custom_target(clone_strings DEPENDS clone_strings)
-else()
-  add_custom_target(clone_strings)
-endif()
+add_custom_target(clone_strings)

 ######################################################################
 # Copy data/ directory target into bin/data/
@@ -181,17 +167,6 @@
   list(APPEND out_data_files ${DATA_OUTPUT_DIR}/${fn})
 endforeach()

-if(GIT_FOUND)
-  # Copy original en.ini to strings.git/en.ini to keep it updated. We
-  # have to manually sync the "en.ini" file in the "strings" repo from
-  # the "aseprite" repo.
-  add_custom_command(
-    OUTPUT ${DATA_OUTPUT_DIR}/strings.git/en.ini
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${SOURCE_DATA_DIR}/strings/en.ini ${DATA_OUTPUT_DIR}/strings.git/en.ini
-    MAIN_DEPENDENCY ${SOURCE_DATA_DIR}/strings/en.ini)
-  list(APPEND out_data_files ${DATA_OUTPUT_DIR}/strings.git/en.ini)
-endif()
-
 add_custom_command(
   OUTPUT ${DATA_OUTPUT_DIR}/README.md
   COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/../README.md ${DATA_OUTPUT_DIR}/README.md
